scripts/test/get_exasol_versions.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+# Ignore shellcheck rule here, as we want to split result by space
+# shellcheck disable=SC2046
+DOCKER_DBS=$(basename -a $(ls -d "${SCRIPT_DIR}/../../docker_db_config_template"/*/))
+
+
+DOCKER_DBS+=($'\ndefault\n')
+DOCKER_DBS+=($'7.1.0-d1\n')
+
+printf '%s' "${DOCKER_DBS[@]}" | jq -R . | jq -cs .
```

Comment:
you may wanna add a newline at the end of the file. 
[This](https://unix.stackexchange.com/a/18789) stack exchange answer elaborates a bit on
the final newline "issue".